### PR TITLE
 docs: Cluster-wide debugging tool documentation

### PR DESCRIPTION
**Pull request by ashwinp**
- Update docs to add a section about the cluster-wide debugging tool.
- Fix some nitpicks in existing documentation; Simplify the content.

Signed-off-by: Ashwin Paranjpe ashwin@covalent.io

*Summary of changes:*

Fixes: https://github.com/cilium/cilium/issues/2349

### CODE REVIEW

File: Documentation/troubleshooting.rst [hunk 1]
```diff
@@ -156,11 +156,11 @@ little bit different
...
 .. Note::
 
-          Please check the dump files for sensitive informations and strip it
+          Please check the dump files for sensitive information and strip it
           away before sharing it with us.
 
-Below is a approximate list of what kind of information is in the archive, but
-you should still verify before sharing.
+Below is an approximate list of the kind of information in the archive. It is
+ recommeded that you verify it before sharing.
```
Comment (aalemayhu) [inline Documentation/troubleshooting.rst]: Remove the space before `recommeded`, otherwise part of the sentence will be bold.

**Review by aalemayhu — APPROVED** [review]
The commit message 5cdf28c56d72 (docs: Cluster-wide debugging tool documentation, 2017-12-18) would look nicer without the bits after the SOB, otherwise LGTM. Thanks.

**Comment by aalemayhu** [comment]
Hmm, missed some things will go through again.

**Review by aalemayhu — CHANGES_REQUESTED** [review]
I was looking at the wrong document earlier, you have several rendering issues which are missing newlines and spaces. The below diff should fix them

```diff
diff --git a/Documentation/troubleshooting.rst b/Documentation/troubleshooting.rst
index 91e92ec1d0d8..6c1aada6dd13 100644
--- a/Documentation/troubleshooting.rst
+++ b/Documentation/troubleshooting.rst
@@ -160,7 +160,7 @@ If you want to capture the archive from a Kubernetes pod, then the process is a
           away before sharing it with us.
 
 Below is an approximate list of the kind of information in the archive. It is
- recommeded that you verify it before sharing.
+recommeded that you verify it before sharing.
 
 * Cilium status
 * Cilium version
@@ -189,14 +189,16 @@ Below is an approximate list of the kind of information in the archive. It is
 
 Cluster-wide Debugging Tool
 ===========================
+
 The ``clusterdebug`` tool can help identify the most commonly encountered
- issues in Cilium deployments. The tool currently supports Kubernetes
- and Minikube clusters only.
+issues in Cilium deployments. The tool currently supports Kubernetes
+and Minikube clusters only.
 
 The tool performs various checks and provides hints to fix specific
 issues that it has identified.
 
 The following is a list of prerequisites:
+
 * Requires Python >= 2.7.*
 * Requires ``kubectl``.
 * ``kubectl`` should be pointing to your cluster before running the tool.
@@ -204,6 +206,7 @@ The following is a list of prerequisites:
 Command to run the cluster-wide debugging tool:
 
 ::
+
     python clusterdebug.zip
 
 Please check the README file for instructions to rebuild the zip file.
```

File: Documentation/troubleshooting.rst (hunk 1 above)
Comment (ashwinp) [reply Documentation/troubleshooting.rst]: Done.

**Comment by ashwinp** [comment]
@scanf Fixed the rendering issues. Please review.

**Review by aalemayhu — APPROVED** [review]

**Review by nebril — APPROVED** [review]

**Pull request merged**